.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
Thanks, but I'd like this template to be separate from this PR. I'm not sure I want to add checklist.